annotations/models.py

```diff
@@ -9,7 +9,7 @@ def labels_pretty(self):
         labels = []
         if self.tense:
             labels.append(self.tense.title)
-        labels.extend(self.labels.values_list('title', flat=True))
+        labels.extend(label.title for label in self.labels.all())
```

Comment:
Does this do something different than before? Does it maybe save some queries?

Reply:
this is another DB optimization, the original code triggers many more queries